src/app/FeedbackForm.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledQuestion = styled.div`
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  color: rebeccapurple;
+`;
+
+const StyledResponse = styled.div`
```

Comment:
you can think about using another element instead of `<div>` for question and response to differentiate. maybe `h3` for question and `<p>` for answer

Reply:
Agreed